dev-python/dpnp/dpnp-0.11.1_rc1.ebuild

```diff
@@ -3,9 +3,8 @@
 
 EAPI=8
 
+DISTUTILS_USE_PEP517=setuptools
 PYTHON_COMPAT=( python3_{10..12} )
-# Breaks library installation
-#DISTUTILS_USE_PEP517=setuptools
```

Comment:
Was this problem fixed? (if not feel free to remove, this package requires DPC++ and that is also currently not building)

Reply:
sys-devel/DPC++-2022.09::science does at least not crash immediately. I am still running this right now (I masked the later one)

Reply:
That one failed to so I treeclean `dpnp`.

Reply:
Masking all of DPC++ (since they all fail) makes more trouble so I will not mask it now.